registry/storage/manifestlisthandler.go

```diff
@@ -101,3 +107,20 @@ func (ms *manifestListHandler) verifyManifest(ctx context.Context, mnfst distrib
 
 	return nil
 }
+
+func (ms *manifestListHandler) platformMustExist(descriptor distribution.Descriptor) bool {
```

Comment:
I'm finding it a bit hard to follow the boolean logic in the flow of `verifyManifest` that calls this `func` but maybe it's the _naming_ of the functions and the boolean logic I must do when reading the the func flow that slightly confuse me.

I thought the `validateImagesExistPlatforms` list would work as follows:

if `validateImagesExistPlatforms` is empty then I'd expect we are saying _no platform_ needs to exist in which case I'd expect the `if len(validateImagesExistPlatforms) == 0` to return `false` which would then lead to `!ms.platformMustExist(manifestDescriptor)` skipping the validation 🤔

Maybe renaming this `func` to `platformSkipExist` and changing the logic would make it easier to read/understand to me personally

Reply:
I think switching it to platformSkipExist would make it even more confusing.

I've added a comment and changed the check to not use continue, which I think helps with clarity. wdyt?